tests/profiles/universal_test.py

```diff
@@ -486,19 +486,59 @@ def test_check_required_tables():
     check = CheckTester(universal_profile,
                         "com.google.fonts/check/required_tables")
 
-    required_tables = ["cmap", "head", "hhea", "hmtx",
+    REQUIRED_TABLES = ["cmap", "head", "hhea", "hmtx",
                        "maxp", "name", "OS/2", "post"]
-    optional_tables = ["cvt ", "fpgm", "loca", "prep",
+
+    OPTIONAL_TABLES = ["cvt ", "fpgm", "loca", "prep",
                        "VORG", "EBDT", "EBLC", "EBSC",
                        "BASE", "GPOS", "GSUB", "JSTF",
-                       "gasp", "hdmx", "kern", "LTSH",
-                       "PCLT", "VDMX", "vhea", "vmtx"]
+                       "gasp", "hdmx", "LTSH", "PCLT",
+                       "VDMX", "vhea", "vmtx", "kern"]
+
     # Our reference Mada Regular font is good here
     ttFont = TTFont(TEST_FILE("mada/Mada-Regular.ttf"))
+    cff_ft = TTFont(TEST_FILE("source-sans-pro/OTF/SourceSansPro-Black.otf"))
+    cff2_ft = TTFont(TEST_FILE("source-sans-pro/VAR/SourceSansVariable-Italic.otf"))
```

Comment:
sure

Reply:
well... it would be even better (following the same style of the rest of the code-tests) if these could actually be `cff_ttFont` and `cff2_ttFont`

Reply:
nah, that's going too far 😀

Reply:
My reasoning here is that it would be easier to read, given that throughout the entire set of code tests we've been using `font` for font file paths and `ttFont` for fontTools TTFont objects.